### PR TITLE
add teasers of the event

### LINKED ISSUE #238: Text to advertise unconference
As discussed on zulip, it might be good to have some sort of "official" text teaser that people can use to advertise the event to their network.

Would be good to have
1) one sentence teaser
2) few sentences teaser
3) one paragraph teaser (Patric has already done one in zulip)

### PR DESCRIPTION
**Pull request by lucaferranti**
fixes #238

### CODE REVIEW

File: content/events/2021-online-unconference/_index.md [hunk 1]
```diff
@@ -89,3 +89,22 @@ We will publish submitted contributions here as soon as they are confirmed.
...
+### Longer teaser
+
+* Are you developing software or tools that are driven by research/engineering in either academia or industry?
+* Need to network, share knowledge and experiences with your peers? 
+* Maybe you have heard of something called research software engineers and you would like to know more? 
+   
+#NordicRSE invites everyone interested in such topics to join our online conference (lightweight get-together) on June 29 and 30, where we let the participants shape the agenda ("birds of a feather"). To kick off the event, we also have four invited speakers, including:
+* one of the Julia programming language core developers, 
+* an R-developer working a an Research Software Engineer (RSE), 
+* one of the developers behind QuTiP (Quantum Toolbox in Python).
```
Comment (lucaferranti) [inline content/events/2021-online-unconference/_index.md:108]: any particular reason for not having the names? These three are confirmed right?

Comment (PatricHolmvall) [reply content/events/2021-online-unconference/_index.md:108]: Yes they are confirmed. By the time of writing, we were still waiting for their consent to publish names, titles/abstracts. Now we have the consent for all three!

File: content/events/2021-online-unconference/_index.md [hunk 2]
```diff
@@ -89,3 +89,22 @@ We will publish submitted contributions here as soon as they are confirmed.
   before, during, and after the event
   
  
+## Share the event
+
+Do you want to share the event with your colleagues/friends? Here are a few texts that you can reuse, feel also free to modify them as you like
+
+### Short teaser
+#NordicRSE invites everyone interested in "Research Software Engineering" activities to join & shape the agenda of the NordicRSE unconference (lightweight get-together) on June 29-30. See [https://nordic-rse.org/events/2021-online-unconference/](https://nordic-rse.org/events/2021-online-unconference/) for more details #RSEng @EOSC_Nordic @coderefine @SciCompAalto
```
Comment (PatricHolmvall) [inline content/events/2021-online-unconference/_index.md:97]: Thanks for adding these suggestions. Small comments: I think we should remove the twitter tags and @s, especially the first #NordicRSE might make it into a heading in markdown. Furthermore, NordicRSE should be written Nordic-RSE.

File: content/events/2021-online-unconference/_index.md (hunk 1 above)
Comment (lucaferranti) [reply content/events/2021-online-unconference/_index.md:108]: should we add the names (and possibly link to their homepage) here?

File: content/events/2021-online-unconference/_index.md (hunk 2 above)
Comment (lucaferranti) [reply content/events/2021-online-unconference/_index.md]: good point! updated

File: content/events/2021-online-unconference/_index.md [hunk 3]
```diff
@@ -89,3 +89,22 @@ We will publish submitted contributions here as soon as they are confirmed.
...
+Nordic-RSE invites everyone interested in "Research Software Engineering" activities to join & shape the agenda of the Nordic-RSE unconference (lightweight get-together) on June 29-30. See [https://nordic-rse.org/events/2021-online-unconference/](https://nordic-rse.org/events/2021-online-unconference/) for more details
+
+### Longer teaser
+
+* Are you developing software or tools that are driven by research/engineering in either academia or industry?
+* Need to network, share knowledge and experiences with your peers? 
+* Maybe you have heard of something called research software engineers and you would like to know more? 
+   
+Nordic-RSE invites everyone interested in such topics to join our online unconference (lightweight get-together) on June 29 and 30, where we let the participants shape the agenda ("birds of a feather"). To kick off the event, we also have four invited speakers, including:
+* one of the Julia programming language core developers,
```
Comment (PatricHolmvall) [inline content/events/2021-online-unconference/_index.md:106]:
```suggestion
* [Kristoffer Carlsson](https://github.com/KristofferC/) (JuliaComputing) _"Julia for research software engineers"_, 
```

File: content/events/2021-online-unconference/_index.md [hunk 4]
```diff
@@ -89,3 +89,22 @@ We will publish submitted contributions here as soon as they are confirmed.
...
+
+### Longer teaser
+
+* Are you developing software or tools that are driven by research/engineering in either academia or industry?
+* Need to network, share knowledge and experiences with your peers? 
+* Maybe you have heard of something called research software engineers and you would like to know more? 
+   
+Nordic-RSE invites everyone interested in such topics to join our online unconference (lightweight get-together) on June 29 and 30, where we let the participants shape the agenda ("birds of a feather"). To kick off the event, we also have four invited speakers, including:
+* one of the Julia programming language core developers, 
+* an R-developer working a an Research Software Engineer (RSE),
```
Comment (PatricHolmvall) [inline content/events/2021-online-unconference/_index.md:107]:
```suggestion
* Athanasia Monica (University of Oslo) _"Developing and distributing in-house R-packages"_, 
```

File: content/events/2021-online-unconference/_index.md [hunk 5]
```diff
@@ -89,3 +89,22 @@ We will publish submitted contributions here as soon as they are confirmed.
...
+### Longer teaser
+
+* Are you developing software or tools that are driven by research/engineering in either academia or industry?
+* Need to network, share knowledge and experiences with your peers? 
+* Maybe you have heard of something called research software engineers and you would like to know more? 
+   
+Nordic-RSE invites everyone interested in such topics to join our online unconference (lightweight get-together) on June 29 and 30, where we let the participants shape the agenda ("birds of a feather"). To kick off the event, we also have four invited speakers, including:
+* one of the Julia programming language core developers, 
+* an R-developer working a an Research Software Engineer (RSE), 
+* one of the developers behind QuTiP (Quantum Toolbox in Python).
```
Comment (PatricHolmvall) [inline content/events/2021-online-unconference/_index.md:108]:
```suggestion
* Shahnawaz Ahmed (Wallenberg Centre for Quantum Technology) _"Keep your code alive - lessons from the QuTiP project"_.
```

File: content/events/2021-online-unconference/_index.md [hunk 6]
```diff
@@ -89,3 +89,22 @@ We will publish submitted contributions here as soon as they are confirmed.
...
+
+### Longer teaser
+
+* Are you developing software or tools that are driven by research/engineering in either academia or industry?
+* Need to network, share knowledge and experiences with your peers? 
+* Maybe you have heard of something called research software engineers and you would like to know more? 
+   
+#NordicRSE invites everyone interested in such topics to join our online conference (lightweight get-together) on June 29 and 30, where we let the participants shape the agenda ("birds of a feather"). To kick off the event, we also have four invited speakers, including:
+* one of the Julia programming language core developers, 
+* an R-developer working a an Research Software Engineer (RSE),
```
Comment (PatricHolmvall) [inline content/events/2021-online-unconference/_index.md]:
```suggestion
* [Athanasia Monica](https://drmowinckels.io/about/) (University of Oslo) _"Developing and distributing in-house R-packages"_, 
```

File: content/events/2021-online-unconference/_index.md (hunk 1 above)
Comment (PatricHolmvall) [reply content/events/2021-online-unconference/_index.md]: Good ideas! Added suggestions for updates.

**Review by PatricHolmvall — APPROVED** [review]

**Pull request merged**